src/main/java/me/leoko/advancedban/manager/CommandManager.java

```diff
@@ -1,5 +1,6 @@
 package me.leoko.advancedban.manager;
 
+import com.imaginarycode.minecraft.redisbungee.RedisBungee;
```

Comment:
Not sure about that but I think this is gonna give a NoClassDefFoundError if the server does not use the RedisBungee as the Scope for this API is set to "provided" in the POM.
Same for all other imports of com.imaginarycode.minecraft.* except for the ones in PubSubMessageListener.java